lib/tasks/sidekiq_metrics.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'aws-sdk-cloudwatch'
+
+namespace :sidekiq do
+  desc 'Send sidekiq queue metrics to Cloudwatch'
+  task send_queue_metrics: :environment do
+    time =  Time.now.utc.iso8601
+
+    cloudwatch = Aws::CloudWatch::Client.new(access_key_id: ENV['AWS_ACCESS_KEY_ID'],
+                                             secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
+                                             region: ENV['AWS_REGION'])
+
+    puts "Getting queues data"
+    queues = Sidekiq::Queue.all.each_with_object({}) do |queue, hash|
+      hash[queue.name] = {
```

Comment:
I'd rather created helper function and created final data in one go. Something like
```
  data.push(build_metric_item(metric_name, value, unit))
```
 But if you like it that way feel free to keep it as it is.